autoload/floaterm/wrapper/ranger.vim

```diff
@@ -6,7 +6,10 @@
 
 function! floaterm#wrapper#ranger#() abort
   let s:ranger_tmpfile = tempname()
-  let cmd = 'ranger ' . $PWD . ' --choosefiles=' . s:ranger_tmpfile
+  let s:original_dir = getcwd()
```

Comment:
It's not necessary to use script-level variable name here, just use `original_dir`

Reply:
Pushed a new commit.